testsuite/tests/index/external-msys2/my_index/index/di/dialog.toml

```diff
@@ -0,0 +1,10 @@
+[general]
+description = "Just for the test"
+maintainers = ["alejandro@mosteo.com"]
+maintainers-logins = ["mosteo"]
```

Comment:
Could this confuse us down the road if I didn't write this test?

Reply:
I just copied from another test. We can use `example@example.com` and `example` instead.